.circleci: Set +u for all conda install commands

seemethere opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #50506 DO NOT MERGE, testing conda builds for py39
* **#50505 .circleci: Set +u for all conda install commands**

Even with +u set for the the conda install it still seems to fail out
with an unbound variable error. Let's try and give it a default value
instead.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

Differential Revision: [D25913692](https://our.internmc.facebook.com/intern/diff/D25913692)

seemethere committed:
.circleci: Attempt to set default value for unbound var

Even with +u set for the the conda install it still seems to fail out
with an unbound variable error. Let's try and give it a default value
instead.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere committed:
Update on ".circleci: Attempt to set default value for unbound var"


Even with +u set for the the conda install it still seems to fail out
with an unbound variable error. Let's try and give it a default value
instead.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

walterddr reviewed (commented):
Did you intend to include tensorpipe upgrade?

seemethere committed:
Update on ".circleci: Set +u for all conda install commands"


Even with +u set for the the conda install it still seems to fail out
with an unbound variable error. Let's try and give it a default value
instead.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere committed:
Update on ".circleci: Set +u for all conda install commands"


Even with +u set for the the conda install it still seems to fail out
with an unbound variable error. Let's try and give it a default value
instead.

Signed-off-by: Eli Uriegas <eliuriegas@fb.com>

[ghstack-poisoned]

seemethere commented:
> Did you intend to include tensorpipe upgrade?

tensorpipe update has been removed
